deno2/.gn

```diff
@@ -17,19 +17,41 @@ secondary_source = "//v8/"
 check_targets = []
 
 default_args = {
+  # Various global chrome args that are unrelated to deno.
+  proprietary_codecs = false
+  safe_browsing_mode = 0
+  toolkit_views = false
+  use_aura = false
+  use_dbus = false
+  use_gio = false
+  use_glib = false
+  use_ozone = false
+  use_udev = false
+
+  # Flags to coax Travis into building.
+  #libcpp_is_static = false
+  use_custom_libcxx = false
+  use_sysroot = false
+
+  use_allocator = "none" # Turn off tcmalloc. Slightly faster builds.
+
   is_component_build = false
   symbol_level = 1
   treat_warnings_as_errors = false
+
+  # https://cs.chromium.org/chromium/src/docs/ccache_mac.md
+  clang_use_chrome_plugins = false
+
   v8_deprecation_warnings = false
   v8_embedder_string = "-deno"
   v8_enable_gdbjit = false
   v8_enable_i18n_support = false
+  v8_experimental_extra_library_files = []
+  v8_extra_library_files = []
   v8_imminent_deprecation_warnings = false
-  v8_monolithic = false
+  v8_monolithic = true
+  v8_static_library=false
```

Comment:
Does this mean we're building a shared library? Or does v8_monolithic turn it into yet something else?

Reply:
Removed. Everything is statically linked in the end - this is inconsequential to our task